feature/experimental/isis/ate_tests/internal/session/session.go

```diff
@@ -256,7 +256,15 @@ func (s *TestSession) PushDUT(ctx context.Context) error {
 		return fmt.Errorf("configuring network instance: %w", err)
 	}
 	dutConf := s.DUTConf.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance).GetOrCreateProtocol(PTISIS, ISISName)
-	_, err := ygnmi.Replace(ctx, s.DUTClient, ProtocolPath().Config(), dutConf)
+
+	// Clear ISIS Protocol
```

Comment:
the purpose of clearing ISIS protocol.

Reply:
Clearing ISIS protocol makes all counters zero, test checks for these counters in beginning of the test.